src/PySXO/__init__.py

```diff
@@ -3,9 +3,7 @@
 from .core.decorators import cache
 from .core.request_handler import RequestHandler
 
-
-__version__ = '0.0.8'
-
+__version__ = '0.0.9'
```

Comment:
IMO this should be in separate PR as we do not have release notes to determine what changes where made between certain versions. It would be nice to at least see this in git log

Reply:
You're technically correct. It *can* be done this way since git will generate a release based on the commit messages. But it's not clean or nice. But ultimately, this is okay unless we need this to become a fully stable independent product. For now, this is okay. But that's a great point and should be noted because you're right